src/Controls/tests/DeviceTests/Memory/MemoryTests.cs

```diff
@@ -83,10 +83,7 @@ void SetupBuilder()
 
 	[Theory("Pages Do Not Leak")]
 	[InlineData(typeof(ContentPage))]
-#if !ANDROID
 	[InlineData(typeof(NavigationPage))]
-	//https://github.com/dotnet/maui/issues/27411
-#endif
 	[InlineData(typeof(TabbedPage))]
 	public async Task PagesDoNotLeak(Type type)
```

Comment:
This one is failing on Android Api Level 23:
<img width="187" alt="image" src="https://github.com/user-attachments/assets/75cc400c-3866-4a2a-ae06-a6e1ff3eba13" />

`Reference to Microsoft.Maui.Controls.NavigationPage (type Microsoft.Maui.Controls.NavigationPage is still alive.\nReference to androidx.fragment.app.FragmentContainerView{38d3432 V.E...... ......ID 0,0-1080,1773 #7f080142 app:id/nav_host} (type AndroidX.Fragment.App.FragmentContainerView is still alive.\n\n---- Assertion timed out`

Reply:
If it works on newer emulators, you might just skip this test on API 23. It probably isn't a leak. There is at least one example of that in here somewhere.